datumaro/tests/test_mot_format.py

```diff
@@ -0,0 +1,146 @@
+            def __iter__(self):
+                return iter([
+                    DatasetItem(id=1, subset='train',
+                        image=np.ones((16, 16, 3)),
+                        annotations=[
+                            Bbox(0, 4, 4, 8, label=2, attributes={
+                                'occluded': True,
+                            }),
+                            Bbox(0, 4, 4, 4, label=3, attributes={
+                                'visibility': 0.4,
+                            }),
+                            Bbox(2, 4, 4, 4, attributes={
+                                'ignored': True
+                            }),
+                        ]
+                    ),
+
+                    DatasetItem(id=2, subset='val',
+                        image=np.ones((8, 8, 3)),
+                        annotations=[
+                            Bbox(1, 2, 4, 2, label=3),
+                        ]
+                    ),
+
+                    DatasetItem(id=3, subset='test',
+                        image=np.ones((5, 4, 3)) * 3,
+                    ),
+                ])
+
+            def categories(self):
+                label_cat = LabelCategories()
+                for label in range(10):
+                    label_cat.add('label_' + str(label))
+                return {
+                    AnnotationType.label: label_cat,
+                }
+
+        class DstExtractor(Extractor):
+            def __iter__(self):
+                return iter([
+                    DatasetItem(id=1,
+                        image=np.ones((16, 16, 3)),
+                        annotations=[
+                            Bbox(0, 4, 4, 8, label=2, attributes={
+                                'occluded': True,
+                                'visibility': 0.0,
+                                'ignored': False,
+                            }),
+                            Bbox(0, 4, 4, 4, label=3, attributes={
+                                'occluded': False,
+                                'visibility': 0.4,
+                                'ignored': False,
+                            }),
+                            Bbox(2, 4, 4, 4, attributes={
+                                'occluded': False,
+                                'visibility': 1.0,
+                                'ignored': True,
+                            }),
+                        ]
+                    ),
+
+                    DatasetItem(id=2,
+                        image=np.ones((8, 8, 3)),
+                        annotations=[
+                            Bbox(1, 2, 4, 2, label=3, attributes={
+                                'occluded': False,
+                                'visibility': 1.0,
+                                'ignored': False,
+                            }),
+                        ]
+                    ),
+
+                    DatasetItem(id=3,
+                        image=np.ones((5, 4, 3)) * 3,
```

Comment:
Why here we don't have attributes at all?

Reply:
`DatasetItem`s do not have `attribute`s yet.